recipe/build.sh

```diff
@@ -1,3 +1,8 @@
 #!/usr/bin/env bash
-make clean
+# Remove previously cythonized files
+find lightning -name \*.cpp -delete
+# Remove PKG-INFO to trigger cythonize
+rm PKG-INFO
+# Remove egg-info because SOURCES.txt contains hard-coded path
+rm -r sklearn_contrib_lightning.egg-info
```

Comment:
Build error otherwise:
```
ValueError: path '/home/pedregosa/anaconda3/lib/python3.8/site-packages/numpy/core/include/numpy/arrayobject.h' cannot be absolute
```
^-- path from `sklearn_contrib_lightning.egg-info/SOURCES.txt`